index.html

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+<head>
+    <link rel="stylesheet" href="styles.css">
+
+    <nav id="navbar" class="nav">
+        <ul class="nav-list">
+          <li>
+            <a href="#welcome-section">About</a>
+          </li>
+          <li>
+            <a href="#projects">Work</a>
+          </li>
+          <li>
+            <a href="#contact">Contact</a>
+          </li>
+        </ul>
+      </nav>
+</head>
+<body>
+    <section id="welcome-section" class="welcome-section">
+        <h1>Hi we are Valentina and Nestor</h1>  
+        <br> Thats our projects</br>  
+    </section>
+
+    <section id="projects" class="projects-section">
+        <h2 class=""projects-section-header">Projects:</h2>
+        <p class="project-title">
+            Project 1
```

Comment:
You need to add at least an _**a**_ element in your #projects

Reply:
- Great Job using inline comments 👍🏽 

- I'd like to kindly suggest that you phrase your comments politely, as suggestions instead of orders.
For example:
This is a polite suggestion: "I think it would be better to add at least an  **a** element in your #projects"

- It's important for you to also state reasons WHY a change needs to implemented.
For example: "I think it would be better to add at least an  **a** element in your #projects because an `<a>` element will . . . ."

- It will be great if you included emojis to make your suggestions sound more friendly. 🤗